docker/pg16/upgrade.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/env bash
+
+set -ex
+set -oi pipefail
+
+# If you need to start over for some reason, destroy the db container and run:
+# docker volume rm hmis-warehouse_dbdata_pg16
+
+docker-compose stop db_previous db
```

Comment:
Shouldn't everybody be using the plugin and running `docker compose` without the dash?

Reply:
For those of us on colima, there's an extra step to get `docker compose` to work without the dash.  That's [documented here](https://github.com/greenriver/hmis-warehouse/pull/4670), so I think we're good dropping it.